cmd/factory.go

```diff
@@ -42,33 +41,22 @@ type Factory interface {
 	ExportRequests() (*lib.ExportRequests, error)
 }
 
-// PathFactory is a function that returns paths to qri & ipfs repos
-type PathFactory func() (string, string)
-
-// EnvPathFactory returns qri & IPFS paths based on enviornment variables
-// falling back to $HOME/.qri && $HOME/.ipfs
-func EnvPathFactory() (string, string) {
-	home, err := homedir.Dir()
-	if err != nil {
-		panic(err)
-	}
-
+// StandardQriPath returns qri paths based on the QRI_PATH environment variable
+// falling back to the default: $HOME/.qri
+func StandardQriPath() string {
 	qriRepoPath := os.Getenv("QRI_PATH")
 	if qriRepoPath == "" {
+		home, err := homedir.Dir()
+		if err != nil {
+			panic(err)
+		}
 		qriRepoPath = filepath.Join(home, ".qri")
 	}
 
-	ipfsFsPath := os.Getenv("IPFS_PATH")
-	if ipfsFsPath == "" {
-		ipfsFsPath = filepath.Join(home, ".ipfs")
-	}
-	return qriRepoPath, ipfsFsPath
+	return qriRepoPath
 }
 
-// NewDirPathFactory creates a path factory that sets qri & ipfs paths to
-// dir/qri & qri/ipfs
-func NewDirPathFactory(dir string) PathFactory {
-	return func() (string, string) {
-		return filepath.Join(dir, "qri"), filepath.Join(dir, "ipfs")
-	}
+// RootedQriPath gives a "/qri" directory from a given root path
+func RootedQriPath(root string) string {
```

Comment:
This is only used in one place, might as well remove it and inline at the call site.